### ISSUE TITLE
TypeError: Cannot set property 'start' of undefined.

### ISSUE THREAD

**Issue by leonov-kirill** [pingyhq/pingy-cli#32]
## Base

- Node.js version: 7.8.0
- NPM version: 4.6.1
- OS: macOS Sierra 10.12.4

## Description
Throws `✖ TypeError: Cannot set property 'start' of undefined.`
**package.json** scripts not added.
**.pingy.json** not created.
Manually created **.pingy.json** and run `node node_modules/@pingy/cli/cli.js dev` works fine.
Tried to install globally – same thing.
## Steps to reproduce
```bash
mkdir my-site
cd my-site
npm i -D @pingy/cli
node node_modules/@pingy/cli/cli.js init
```
## Result
```
0.6.6 (local)
  ____  _                   
 |  _ \(_)_ __   __ _ _   _ 
 | |_) | | '_ \ / _` | | | |
 |  __/| | | | | (_| | |_| |
 |_|   |_|_| |_|\__, |\__, |
                |___/ |___/ 


? What document format do you wish to use HTML
? What styles format do you wish to use Sass (.sass)
? What scripts format do you wish to use Babel
? Choose the folder name to export compiled files to public
✖ TypeError: Cannot set property 'start' of undefined
? Do you want Pingy to scaffold the following files for you? Yes
? The most important question: Tabs or spaces Tabs
✔ Site files scaffolded

Ready to install dependencies.

Command that will now be run:
  > npm install --save-dev @pingy/cli node-sass babel-core babel-preset-env

? Run this command now? Yes
...
✔ Dependencies installed
```
##

**Comment by davej** [comment]
Ah, Pingy expects a `scripts` object in any existing `package.json`. I will get this fixed in a few weeks until then you can either ignore this error (as it won't cause a problem if you don't use npm scripts), or you could add an empty `scripts` object (`scripts: {}`) to your `package.json`.

**Comment by leonov-kirill** [comment]
Actually, I already created [default .pingy.json configuration file](https://github.com/pingyhq/pingy-cli#configuring-export) and added scripts manually like this:
```json
...
"scripts": {
    "dev": "node_modules/@pingy/cli/cli.js dev",
    "export": "node_modules/@pingy/cli/cli.js export"
  }
...
```
So everything works excellent.

P.S. Thanks for fast reply and such awesome tool it makes front-end a breeze.

**Comment by davej** [comment]
Cool, npm scripts actually look in your local bin by default so the following is the same and will use the local version:

```
...
"scripts": {
    "dev": "pingy dev",
    "export": "pingy export"
  }
...
```

**Comment by leonov-kirill** [comment]
Wow, thanks!

**Comment by davej** [comment]
Fixed in [0.6.9](https://github.com/pingyhq/pingy-cli/releases/tag/%40pingy%2Fcli%400.6.9)
